components/map/MapForPrint.vue

```diff
@@ -1,233 +1,450 @@
+                    <div style="width: 25%">
+                        <v-img contain src="/img/logocmr_normal_min.png" />
                     </div>
-                </l-control>
-
-                <l-control-scale
-                    v-if="user.settings.map_scale_visible"
-                    position="bottomleft"
-                />
-
-                <l-geo-json
-                    ref="interestArea"
-                    :geojson="interestArea"
-                    :options-style="interestStyle"
-                    :visible="showInterestArea"
-                />
-
-                <SupportLayers />
-
-                <MonitoringLayers v-if="!monitoringGeoserver" :map="map" />
-
-                <BaseWmsMetadataPopup :map="map" />
-
-                <PriorityLayers :map="map" />
-            </l-map>
-
-            <div v-if="loading" class="loading-background">
+                </div>
+                <div class="text-center font-weight-bold">
+                    <p class="font-weight-bold mb-1">
+                        {{ titleMap }}
+                    </p>
+                    <p class="mb-2">
+                        Centro de Monitoramento Remoto - Fundação Nacional do
+                        Índio
+                    </p>
+                </div>
+                <div
+                    class="d-flex justify-center"
+                    style="height: 150px; max-height: 150px; width: 100%"
+                >
+                    <client-only>
+                        <l-map
+                            ref="miniPrintMap"
+                            id="miniPrintMap"
+                            :zoom="zoomMiniMap"
+                            :options="optionsMiniMap"
+                        >
+                            <l-tile-layer
+                                url="http://{s}.tile.osm.org/{z}/{x}/{y}.png"
+                                :attribution="attribution"
+                            ></l-tile-layer>
+                            <l-control position="topleft" class="ma-0 pa-0">
+                                <p class="ma-1 pa-0 print-mini-map-text">
+                                    LOCALIZAÇÃO DA ÁREA
+                                </p>
+                            </l-control>
+                        </l-map>
+                    </client-only>
+                </div>
                 <div>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
-                    <v-skeleton-loader
-                        type="chip"
-                        width="32"
-                    ></v-skeleton-loader>
+                    <p class="d-block ma-1">Legenda:</p>
+                    <div class="ma-1 flex-wrap" style="width: 100%">
```

Comment:
inline style? Why dont you use vue style?

Reply:
Used inline because I don't see the need to create several classes just to define the width of the container. The lib (domtoimage) generates the image according to the size of the elements so it was also necessary to define a fixed width.